core/google/cloud/credentials.py

```diff
@@ -30,58 +30,9 @@
 def get_credentials():
     """Gets credentials implicitly from the current environment.
 
-    .. note::
```

Comment:
Indeed. If we don't want users to use it, let's remove it from the public interface (file a bug?)